src/scenes/home/ourPrograms/mentorshipProgram/mentorshipProgram.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { FaCheck, FaBan } from 'react-icons/lib/fa';
+import PropTypes from 'prop-types';
+import styles from './mentorshipProgram.css';
+
+const MentorshipProgram = ({ fontSize }) => (
+  <div>
+    <div className={styles.flexContainer}>
+      <div className={styles.featureHeading}>
+        <FaCheck size={fontSize} />
+        <p>What It Is</p>
+      </div>
+      <div className={styles.featureDescription}>
+        <ul>
+          <li>
+            A set of scheduled phone calls or video conferences aimed at having professional
+            developers guide your next steps.
+          </li>
+          <li>
+            Friendly monitoring from a volunteer staff member, encouraging you to keep working
+            towards your goal.
+          </li>
+          <li>An environment of friendly support from developers of many experience levels.</li>
+          <li>A phenomenal tool for growth in your journey into the tech industry.</li>
+        </ul>
+      </div>
+    </div>
+    <div className={styles.flexContainer}>
+      <div className={styles.featureHeading}>
+        <FaBan size={fontSize} />
+        <p>What It Isn&apos;t</p>
+      </div>
+      <div className={styles.featureDescription}>
+        <ul>
+          <li>
+            A live &quot;Ask Jeeves&quot; alternative. Mentors are happy to help you, but also
+            expert a certain degree of individual effort. Please spend time{' '}
+            <a
+              href="https://community.operationcode.org/t/the-art-of-asking-questions/121"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              learning how to ask questions
+            </a>{' '}
+            and Googling for yourself before going to our volunteers for aid.
+          </li>
+          <li>
+            A guarantee that someone will work with you long-term. All our mentors are volunteers
```

Comment:
I think we want to do this a bit differently (cc @jjhampton @hpjaj @Hiren-Mehta) - we should set the expectation of at least having a 30-minute session with someone. Whether that commitment is long-term kinda depends on whether the mentor-mentee relationship is the right fit.

Something to note for your mentorship efforts also @dshaw